r/tests/testthat/test-arrow.R

```diff
@@ -76,4 +77,4 @@ test_that("MemoryPool calls gc() to free memory when allocation fails (ARROW-100
   # but it should gc() and retry (and fail again)
   expect_error(BufferOutputStream$create(2**60))
   expect_true(env$gc_was_called)
-})
+})
```

Comment:
```suggestion
})

```

To appease the lintr, styler _should_ do this, but this is easy enough.

Reply:
Yeah this is a case of vscode sometimes stripping the trailing newline